projects/components/src/table/table.component.ts

```diff
@@ -692,15 +696,18 @@ export class TableComponent
     this.pageChange.emit(pageEvent);
   }
 
-  private getPageData(params: ParamMap): Partial<PageEvent> | undefined {
+  private getPagination(params: ParamMap): Partial<PageEvent> | undefined {
```

Comment:
```suggestion
  private getPagination(params: ParamMap): Partial<PageEvent> {
```

can never emit undefined any more